kucher/__init__.py

```diff
@@ -17,7 +17,7 @@
 import sys
 
 if sys.version_info[:2] < (3, 6):
-    raise ImportError('A newer version of Python is required')
+    raise ImportError("A newer version of Python is required")
```

Comment:
I suggest we delete this as python3.10 is used in all modern OSes. Or at least we bump the version number to 3.10.